posting/writer_test.go

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 Dgraph Labs, Inc. and Contributors
```

Comment:
`badger 2020/03/30 18:26:27 INFO: All 0 tables opened in 0s
goos: linux
goarch: amd64
pkg: github.com/dgraph-io/dgraph/posting
BenchmarkTxnWriter-8    	      13	 233417758 ns/op
BenchmarkWriteBatch-8   	      27	  94321812 ns/op
PASS
ok  	github.com/dgraph-io/dgraph/posting	6.102s`

Benchmark results for txnWriter versus WriteBatch